templates/lune/data.json

```diff
@@ -8,7 +8,7 @@
     },
     "navbar": {
       "name": "Lune",
-      "svg-trigger": "<svg xmlns='http://www.w3.org/2000/svg' width='36' height='36' viewBox='0 0 36 36' class='block'><path fill='currentColor' d='M9 12h18v2H9zM9 17h18v2H9zM9 22h18v2H9z'/></svg>",
+      "svg-trigger": "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 36 36' width='36' height='36'><style>.st{fill:currentColor;}</style><path class='st' d='M9 12h18v2H9z'/><path class='st' d='M9 17h18v2H9z'/><path class='st' d='M9 22h18v2H9z'/></svg>",
```

Comment:
Changed the SVG trigger SVG so paths are separated to allow us to animate them separately.